sympy/polys/numberfields/minpoly.py

```diff
@@ -18,7 +18,7 @@
 from sympy.utilities.iterables import subsets
 
 from sympy.polys.domains import ZZ, QQ, FractionField
-from sympy.polys.orthopolys import dup_chebyshevt
+from sympy.polys.orthopolys import dup_chebyshev
```

Comment:
I think the `dup_chebyshevt` and `dup_chebyshevu` functions should be preserved (although they could both call a new `dup_chebyshev` function).

Reply:
Both fixed.